authenticationsdk/util/MLEUtility.py

```diff
@@ -1,21 +1,140 @@
+import json
+import time
+from datetime import datetime, timezone
+
+from cryptography import x509
+from jwcrypto import jwk, jwe
+
+from authenticationsdk.util.Cache import FileCache
+from authenticationsdk.util.GlobalLabelParameters import GlobalLabelParameters
+import CyberSource.logging.log_factory as LogFactory
 
 class MLEUtility:
+    logger = None
+
+    @staticmethod
+    def setup_logger(merchant_config):
+        MLEUtility.logger = LogFactory.setup_logger(__name__, merchant_config.log_config)
+
+    class MLEException(Exception):
+        def __init__(self, message, errors=None):
+            super().__init__(message)
+            self.errors = errors
+
     @staticmethod
-    def check_is_mle_for_api(merchant_config, isMLESupportedByCybsForApi, operationIds):
```

Comment:
Why this function has changes?
Also, there is no need to for logger in this function.